nautobot/core/views/generic.py

```diff
@@ -101,6 +103,19 @@ def get(self, request, *args, **kwargs):
         Generic GET handler for accessing an object.
         """
         instance = get_object_or_404(self.queryset, **kwargs)
+        # Get the ObjectChange records to populate the advanced tab information
+        object_change_records = get_changes_for_model(instance)
+        created_by = None
+        last_updated_by = None
+        try:
+            created_by_record = object_change_records.get(action=ObjectChangeActionChoices.ACTION_CREATE)
+            created_by = created_by_record.user
+        except ObjectChange.DoesNotExist:
+            pass
+
+        last_updated_by_record = object_change_records.filter(action=ObjectChangeActionChoices.ACTION_UPDATE).first()
```

Comment:
Wonder if the `filter` is actually needed here. If the newest record is a `Delete` we'll never see this page, and if it was a `Create` only, then that's still the "last updated" information for this model?